src/Base/Unit.cpp

```diff
@@ -508,6 +508,16 @@ QString Unit::getTypeString() const
         return QString::fromLatin1("KinematicViscosity");
     if (*this == Unit::VacuumPermittivity)
         return QString::fromLatin1("VacuumPermittivity");
+    if(*this == Unit::VolumeFlowRate              )
```

Comment:
Minor but nonetheless: please follow the existing formatting of a file. This might seem nitpicking but later the automated clang formatter will produce unnecessary diffs when someone change s the code in future.

Reply:
No, I agree; this is exactly the reason I used this funny formatting in the first place. However the formatting of the file was changed in the interim, and I missed this when rebasing; apologies.